packages/theme-default/src/layout/DocLayout/index.tsx

```diff
@@ -74,7 +74,7 @@ export function DocLayout(props: DocLayoutProps) {
       <div
         className={`${styles.content} rspress-doc-container flex flex-shrink-0 mx-auto`}
       >
-        <div className="w-full overflow-hidden flex-1">
+        <div className="w-full">
```

Comment:
Can we remove `flex-1`? It seems still useful:

![image](https://github.com/web-infra-dev/rspress/assets/7237365/6aeb21ff-8bfd-4ba0-b44d-e1204495110c)

Reply:
We cannot remove the `flex-1` here, this property is crucial to ensure that the width of the outline always remains at 268px. Without this property, the widths of the container and outline would be redistributed due to flex.

Reply:
I get it